### ISSUE TITLE
Funny Particles when making Livisite Alloy in the Create Mixer

### ISSUE THREAD

**Issue by GlintingPlanet11** [NyagiByte/Compression#59]
![2024-10-28_20 06 13](https://github.com/user-attachments/assets/ce87bd99-9776-44fe-a5e3-4010b0006090)
Missing Texture Particles when making Livisite Alloy in the Create Mixer.
Version: 1.1b

**Comment by NyagiByte** [comment]
Yeah i'm not really sure what causes the missing texture with these, i'll have to look into it

**Comment by GlintingPlanet11** [comment]
Also happens when making brass

**Comment by jjfireblast101** [comment]
This is a texture issue with chemlib I'll take a looksie at some point
